pkg/operator/ceph/cluster/rbd/mirror_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbd
+
+import (
+	"testing"
+
+	cephv1beta1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1beta1"
+	rookalpha "github.com/rook/rook/pkg/apis/rook.io/v1alpha2"
+	"github.com/rook/rook/pkg/clusterd"
+	testop "github.com/rook/rook/pkg/operator/test"
+	exectest "github.com/rook/rook/pkg/util/exec/test"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestRBDMirror(t *testing.T) {
+	clientset := testop.New(1)
+	keysCreated := map[string]bool{}
+	executor := &exectest.MockExecutor{}
+	executor.MockExecuteCommandWithOutputFile = func(debug bool, actionName string, command string, outFileArg string, args ...string) (string, error) {
+		logger.Infof("%s %+v", command, args)
+		if args[0] == "auth" && args[1] == "get-or-create-key" {
+			daemonName := args[2]
+			keysCreated[daemonName] = true
+			return "{\"key\":\"mysecurekey\"}", nil
+		}
+		return "", nil
+	}
+
+	c := New(&clusterd.Context{Clientset: clientset, Executor: executor},
```

Comment:
Please move the first arg to its own line as the other args.

Reply:
we have some flexibility here since gofmt doesn't enforce it. my preference is actually like this. it looks awkward to me if `New(` is by itself.